examples/lb3-application/lb3app/server/boot/create-sample-models.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const debug = require('debug')('loopback:example:lb3application');
+
+module.exports = function(app, cb) {
+  app.dataSources.db.automigrate('CoffeeShop', function(err) {
```

Comment:
Similarly here, I'd prefer to use `await` instead of callbacks. I understand this code is imported from an existing LB3 example app, thus if you don't want to change it, then I would not mind too much.

```ts
module.exports = async function(app) {
  await app.dataSources.db.automigrate('CoffeeShop');
  const coffeeShops = await app.models.CoffeeShop.create([
    // ...
  ]);

  debug('Models created: \n', coffeeShops);
};
```